packages/clerk-js/src/ui/components/PricingTable/PlanCard.tsx

```diff
@@ -13,44 +13,35 @@ import {
   SimpleButton,
   Span,
   Text,
-  useAppearance,
 } from '../../customizables';
 import { Avatar, SegmentedControl } from '../../elements';
-import { usePrefersReducedMotion } from '../../hooks';
-import { Check, InformationCircle, Minus, Plus } from '../../icons';
+import { useMotionSafe } from '../../hooks';
+import { Check, InformationCircle, Plus, Minus } from '../../icons';
```

Comment:
I think the linter is going to yell at you about the order of these
```suggestion
import { Check, InformationCircle, Minus, Plus } from '../../icons';
```